setup.py

```diff
@@ -4,7 +4,7 @@
 
 from setuptools import setup, find_packages
 
-UTILS_VERSION = "5114b38b4bca476e2312035226b9a5a65b5c2cdb"
+UTILS_VERSION = "52a7ce3dbe9b96a6a09e9e9ce029b2840a9eb937"
```

Comment:
```suggestion
UTILS_VERSION = "21a1d4b552e2a821f9414bd9a35377e5892a260b"
```
Update to latest minware-singer-utils version.
**this is required, or there will be a regression in tap-azure-git**